src/test/java/org/opentripplanner/middleware/typeform/TypeFormDispatcherTest.java

```diff
@@ -0,0 +1,24 @@
+package org.opentripplanner.middleware.typeform;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class TypeFormDispatcherTest {
+
+    @Test
+    void canMakeResponsesParams() {
+        // October 25, 2024 00:00, US Pacific Daylight Time
```

Comment:
In order to use the config param `OTP_TIMEZONE: America/Los_Angeles` and for this test to pass (outside of this time zone), the following lines have to be added to this test:

`// Load default env.yml configuration.`
`ConfigUtils.loadConfig(new String[]{});`

Reply:
You are right, the class can actually extend `OTPMiddlewareTestEnvironment` that includes loading the config (https://github.com/ibi-group/otp-middleware/pull/267/commits/18ecd9fb882c1b0ecc87b243335394f4d9bbb23f).